### ISSUE TITLE
Vivaldi

### ISSUE THREAD

**Issue by Thomas-QM** [spikespaz/search-deflector#5]
After setting up Search Deflector (which successfully recognized Vivaldi), I set the edge protocol to launcher.exe, and when I search it appears to load the program, which does nothing with Vivaldi..

Not sure what other info to include other than Vivaldi's diagnostic info.
![system info](https://user-images.githubusercontent.com/13261878/44633323-bab09000-a93e-11e8-9f1d-b16f79e3263c.png)

**Comment by spikespaz** [comment]
Were you able to select Vivaldi in the setup and was the path correct? If you don't remember, open your start menu, and in the apps list look for the "Search Deflector" folder. Expand it, and click the "Configure" shortcut. Then tell me about each step and what's printed in the console.

If Vivaldi is your default browser, press Windows+R and type `https://github.com` and press Enter. Does it take you to GitHub? Now, press Windows+R again. Type `microsoft-edge://github.com` and press Enter. What now?

Find the Vivaldi executable, press Shift+Right Click, and open a new PowerShell window. Then enter the command `vivaldi.exe "https://github.com"`. Again, are you at GitHub?

Make sure Vivaldi is your system default browser, and try running the setup again. This time, select the "System Default" option for the browser. Try it again, does it work now?

**Comment by Thomas-QM** [comment]
wait why did i talk in proper grammar

so yeah it shows up, path correct
...ok just selected system default it works

starting it from cmd also works fine

tldr: it doesn't work when selecting vivaldi, but system default does

**Comment by spikespaz** [comment]
Did all of the things I asked you to do above work?

I need to know what happens after you perform each of the four actions listed above.

**Comment by Thomas-QM** [comment]
um so like i said, 1 works
2 doesnt (sorry forgot to mention lol), 3 & 4 do
and the url only works if i configure search deflector with system default instead of vivaldi
imo sounds like a process problem
